blockchain/blockchain_test.go

```diff
@@ -443,125 +443,6 @@ func TestState(t *testing.T) {
 	})
 }
 
-func TestEvents(t *testing.T) {
```

Comment:
Is this test obsolete?

Reply:
I moved it into a possibly better place
https://github.com/NethermindEth/juno/blob/c391201aba707d33fa7f7ae4315856a0ca3d26a0/rpc/events_test.go#L27